webapp/channels/src/components/team_controller/team_controller.tsx

```diff
@@ -26,6 +25,7 @@ import type {OwnProps, PropsFromRedux} from './index';
 
 const BackstageController = makeAsyncComponent('BackstageController', lazy(() => import('components/backstage')));
 const Pluggable = makeAsyncComponent('Pluggable', lazy(() => import('plugins/pluggable')));
+const ChannelController = makeAsyncComponent('ChannelController', lazy(() => import('components/channel_layout/channel_controller')));
```

Comment:
This is another one that I don't think should be lazily loaded

Reply:
reverted